src/utils/createTemplate.js

```diff
@@ -0,0 +1,19 @@
+/**
+   *  Create Email Template
+   *
+   * @param {object} template - template object containing text(string) and html(string)
+   * @param {string} link - link to embed in template
+   *
+   * @returns {object} returns object containing text(formatted) and html(formatted)
+   *
+   * @example
+   *
+   *    createTemplate(resetPasswordTemplate, 'www.resetpassword.com')
+  */
+const createTemplate = (template, link) => {
+  const { text } = template(link);
+  const { html } = template(link);
```

Comment:
Is it possible to destructure `text` and `html` like so `const { text, html } = template(link)`

Reply:
Done. Thanks, that's actually better

Reply:
Yes